lib/rack/lint.rb

```diff
@@ -356,6 +356,15 @@ def check_environment(env)
         unless env[SCRIPT_NAME] != "/"
           raise LintError, "SCRIPT_NAME cannot be '/', make it '' and PATH_INFO '/'"
         end
+
+        ## <tt>rack.response_finished</tt>:: An array of callables run after the HTTP response has been sent.
```

Comment:
Should there be some mention of upstream servers implementing this functionality similar to `rack.hijack`? How much do we want to leave up to the implementation?

Reply:
My feeling is, if we care about this feature, it should be a mandatory requirement of Rack 3. However, maybe @tenderlove can give his opinion?